ocaml/idl/dm_api.ml

```diff
@@ -225,7 +225,10 @@ let check api emergency_calls =
   (* Sanity check 7: message parameters must be in increasing order of in_product_since *)
   let are_in_vsn_order ps =
     let rec getlast l = (* TODO: move to standard library *)
-      match l with [x] -> x | _::xs -> getlast xs | [] -> raise (Invalid_argument "getlast") in
+      match l with
+      | [x] -> x.code_name
```

Comment:
I think it's not a good idea to mix a standard function (`getlast` - usually called `last`) with something specific here. I'd keep `getlast` and work with the result at the call site:

```
getlast xs |> fun x -> x.code_name
```